docs/content/installation/using-gcp-marketplace-package.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Using GCP Marketplace Ingress Controller
+description: "This document will walk you through the steps needed to deploy the NGINX Ingress Controller through the GCP Marketplace."
+weight: 2400
+doctypes: [""]
+toc: true
+docs: "DOCS-608"
```

Comment:
```suggestion
docs: "DOCS-000"
```
The doc codes are used for monitoring the state of documents: 608 corresponds to the following page.

https://docs.nginx.com/nginx-ingress-controller/installation/using-the-jwt-token-docker-secret/